### PR TITLE
Picture preview when replying. 

### LINKED ISSUE #2916: Picture preview when replying
It's very unclear which picture you're replying to when you reply to a picture in a chat because it is only a link, with no timestamp and no preview (two things most other messaging clients use). Having a miniaturized picture would help a lot to clarify this without a bunch of extra clicking, or at least, having the entire message be able to be tapped to see the message in question rather than just a small link on the left.

### PR DESCRIPTION
**Pull request by bmarty**
Also add the image preview in the message detail bottomsheet

Fixes #2916

#### Reply

image

#### Bottomsheet

image

### CODE REVIEW

File: vector/src/main/java/im/vector/app/core/epoxy/bottomsheet/BottomSheetMessagePreviewItem.kt [hunk 1]
```diff
@@ -59,6 +67,10 @@ abstract class BottomSheetMessagePreviewItem : VectorEpoxyModel<BottomSheetMessa
         holder.avatar.setOnClickListener { userClicked?.invoke() }
         holder.sender.setOnClickListener { userClicked?.invoke() }
         holder.sender.setTextOrHide(matrixItem.displayName)
+        data?.let {
+            imageContentRenderer?.render(it, ImageContentRenderer.Mode.THUMBNAIL, holder.imagePreview)
```
Comment (BillCarsonFr) [inline vector/src/main/java/im/vector/app/core/epoxy/bottomsheet/BottomSheetMessagePreviewItem.kt:71]: clear on unbind?

**Review by BillCarsonFr — APPROVED** [review]
just one remark

File: vector/src/main/java/im/vector/app/core/epoxy/bottomsheet/BottomSheetMessagePreviewItem.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/core/epoxy/bottomsheet/BottomSheetMessagePreviewItem.kt:71]: Done

**Comment by kojid0** [comment]
Quick question: When you reply to a photo (swipe the photo, then enter some text and hit send), it will still say "In reply to [Name] sent an image.", right? So no picture preview here? This seems weird to me. If this is intended, okay. Otherwise I will open a new issue.

**Pull request merged**